### DISCUSSION TITLE
posts/%E8%A8%AD%E5%AE%9AUbuntu-NTP-%E6%A0%A1%E6%99%82/

### DISCUSSION THREAD

**Discussion by giscus** [jenhaoyang/jenhaoyang.github.io#3] [Announcements]
# posts/%E8%A8%AD%E5%AE%9AUbuntu-NTP-%E6%A0%A1%E6%99%82/

timedatectl 時間管理工具 顯示目前的設定狀態 1 2 # 顯示目前狀態 timedatectl

https://jenhaoyang.github.io/posts/%E8%A8%AD%E5%AE%9AUbuntu-NTP-%E6%A0%A1%E6%99%82/

**Comment by jenhaoyang** [comment]
test comment

**Comment by jenhaoyang** [comment]
aaa
